components/logins/src/db.rs

```diff
@@ -307,6 +308,57 @@ impl LoginDb {
         rows.collect::<Result<_>>()
     }
 
+    pub fn get_by_base_domain(&self, base_domain: &str) -> Result<Vec<Login>> {
+        // We just do a linear scan for now. Later we might choose to have an
```

Comment:
Mainly asking this for my own edification, do you anticipate the need for future optimization here? I'm assuming based on the few metrics I have seen that even in a linear scan in this scenario won't have an adverse impact, but you may know something I don't.

Reply:
I would argue against implementing this optimization tbh, it would cause a bunch of complexity and record counts for logins are small. For stuff like history we do it because you have hundreds of thousands of records, and thus a table scan is terrible.